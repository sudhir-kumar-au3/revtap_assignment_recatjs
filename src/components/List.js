import React from 'react'

function List(props) {
    return (
        <>
            <tr>
                <td>{props.id}</td>
                <td>{props.firstName}</td>
                <td>{props.lastName}</td>
                <td>{props.email}</td>
                <td>{props.created}</td>
                <td>{props.orders}</td>
            </tr>
         </>
    )
}

export default List
