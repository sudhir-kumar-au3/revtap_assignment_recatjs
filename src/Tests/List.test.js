import React from "react";
import renderer from "react-test-renderer";
import List from "../components/List";

it("renders correctly", () => {
  const tree = renderer.create(<List></List>).toJSON();
  expect(tree).toMatchSnapshot();
});
it("renders correctly one item", () => {
  const tree = renderer
    .create(
      <List
        id={"some random id"}
        firstName={"sudhir"}
        lastName={"kumar"}
        created={"any date"}
        orders={3}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
