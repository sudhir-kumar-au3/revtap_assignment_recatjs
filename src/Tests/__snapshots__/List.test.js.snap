// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<tr>
  <td />
  <td />
  <td />
  <td />
  <td />
  <td />
</tr>
`;

exports[`renders correctly one item 1`] = `
<tr>
  <td>
    some random id
  </td>
  <td>
    sudhir
  </td>
  <td>
    kumar
  </td>
  <td />
  <td>
    any date
  </td>
  <td>
    3
  </td>
</tr>
`;
